tests/wallet.rs

```diff
@@ -336,7 +336,8 @@ mod node {
             .into_iter()
             .map(|value| {
                 let obfuscated = (rng.next_u32() & 1) == 1;
-                let idx = rng.next_u64() % MAX_KEY as u64;
+                let idx = rng.next_u64()
+                    % (if MAX_KEY == 0 { 1 } else { MAX_KEY }) as u64;
```

Comment:
This check is useless, `MAX_KEY` should never be `0`, if someone set `0` by mistake it's totally fine that the test would fail because of that. Please, put back the previous code.